.yamllint

```diff
@@ -0,0 +1,26 @@
+# SPDX-License-Identifier: Apache-2.0
+
+extends: default
+
+ignore: |
+  /external
+  /docs/doxygen/doxygen-awesome-css
+  mtb_shared
+
+rules:
+  line-length:
+    max: 100
+    ignore: |
+      /.github/actions/safe-upload-artifacts/action.yml
```

Comment:
Why ignore these files?

Reply:
There is also https://yamllint.readthedocs.io/en/stable/disable_with_comments.html which we could use instead of ignoring whole file.

Reply:
Rechecked and seems that those issues with >100 columns could be solved in a reasonable way.